app/controllers/physical_rack_controller.rb

```diff
@@ -0,0 +1,47 @@
+class PhysicalRackController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::MoreShowActions
+
+  before_action :check_privileges
+  before_action :session_data
```

Comment:
I think here the `get_` prefix is preferred, consistency beats rubocop.

Reply:
:eye: :eagle: